packages/smithy-http/src/smithy_http/aio/crt.py

```diff
@@ -193,6 +195,18 @@ def _cancel(self, completion_future: ConcurrentFuture[int | Exception]) -> None:
             self._response_future.cancel()
 
 
+class CRTErrorResponse(Exception):
```

Comment:
We want all exceptions in the library to inherit from a top level Smithy/AWS exception to allow customers to be able to grab everything.

```suggestion
class CRTErrorResponse(SmithyHTTPException):
```

Reply:
This should never be raised to customers (its internal to the CRT client) - should we still use the Smithy exception?

Reply:
Edit: With Jordon's suggestion below this exception is no longer used at all.